scripts/ci

```diff
@@ -3,11 +3,10 @@ set -e
 
 cd $(dirname $0)
 
-./build
+./validate-ci
```

Comment:
this needs to be kept at the end. It's a validation that none of the other scripts that are executed produce artifacts after execution

Reply:
reverted